docs/contributing/community.md

```diff
@@ -1,91 +1,75 @@
 ---
-title: Community
+title: Comunidade
 ---
 
-As of September 2019, the Gatsby community is comprised of over 2,400 contributors and tens of thousands of developers building both personal and commercial websites on Gatsby.
+Desde Setembro de 2019, a comunidade Gatsby é composta por mais de 2.400 colaboradores e dezenas de milhares de desenvolvedores construindo websites pessoais e comerciais usando Gatsby.
 
-Open source doesn’t always have the best reputation for being friendly and welcoming, and that makes us sad. **Everyone belongs in open source, and Gatsby is dedicated to making you feel welcome.**
+_Open source_ nem sempre tem a melhor reputação de ser amigável e acolhedor, e isso nos deixa tristes. **Todos que pertencem ao _open source_, e todos do Gatsby estão dedicados a fazer você se sentir acolhido.**
 
-We will never judge, condescend, or exclude anyone. Instead, we will go above and beyond to support the community, through helping you contribute to the Gatsby ecosystem, offering [free swag for contributors](https://gatsby.dev/swag), giving control to the community by [auto-inviting all contributors to the Gatsby GitHub org](https://github.com/gatsbyjs/gatsby/pull/7699#issuecomment-416665803), an open and inclusive [code of conduct](/contributing/code-of-conduct/), and other means that empower and embrace the incredible community that makes Gatsby possible.
+Nós nunca julgaremos, seremos condescendentes ou excluiremos alguém. Ao invés disso, nós iremos além para dar suporte à comunidade, ajudando você a contribuir com o ecossistema do Gatsby, oferecendo [brindes para os colaboradores](https://gatsby.dev/swag), dando controle para a comunidade [ao convidar automaticamente todos os colaboradores da organização Gatsby no GitHub](https://github.com/gatsbyjs/gatsby/pull/7699#issuecomment-416665803), um [código de conduta](/contributing/code-of-conduct/) aberto e inclusivo, e outros meios que capacitam e abraçam a incrível comunidade que faz o Gatsby ser possível.
 
-One of our community's values is that ["you belong here"](/blog/2018-09-07-gatsby-values/#you-belong-here).
+Um dos valores da nossa comunidade é que ["seu lugar é aqui"](/blog/2018-09-07-gatsby-values/#you-belong-here).
 
-## Gatsby Contributor Covenant Code of Conduct
+## Código de Conduta do Colaborador Gatsby
```

Comment:
Segundo o guia de tradução do WordPress, "no inglês, a regra de maiúsculas e minúsculas é diferente da usada no português. Em português apenas a primeira palavra da frase e os nomes próprios devem ter a primeira letra maiúscula."

Em alguns títulos você seguiu essa regra e em outros não.

Reply:
Os títulos estão seguindo o padrão [Title Captalization](https://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html). Alguns estavam fora deste padrão, mas já foram corrigidos.

Reply:
Acho que deveríamos seguir a regra do português (letra maiúscula apenas na primeira palavra ou em nomes próprios). Mas seria bom ver o que os outros revisores acham.